skimage/restoration/_denoise.py

```diff
@@ -548,15 +558,34 @@ def denoise_wavelet(img, sigma=None, wavelet='db1', mode='soft',
     """
     img = img_as_float(img)
 
+    if type(sigma) in {float, int, complex}:
+        sigma = [sigma] * img.shape[-1]
+
     if multichannel:
         out = np.empty_like(img)
         for c in range(img.shape[-1]):
             out[..., c] = _wavelet_threshold(img[..., c], wavelet=wavelet,
-                                             mode=mode, sigma=sigma,
+                                             mode=mode, sigma=sigma[c],
                                              wavelet_levels=wavelet_levels)
+    elif convert2ycbcr:
```

Comment:
"explicit is better than implicit". This way it is clearer for the user, and is easier for the developer to be refactored/deprecated in case of any architectural changes.